sqlalchemy_sphinx/dialect.py

```diff
@@ -50,14 +61,15 @@ def visit_options_func(self, fn, *args, **_kw):
     def limit_clause(self, select, **kw):
         text = ""
         if select._limit is not None and select._offset is None:
-            text += "\n LIMIT 0, {}".format(select._limit)
+            text += "\n LIMIT 0, {0}".format(select._limit)
         else:
             text += "\n LIMIT %s, %s" % (
                 self.process(sql.literal(select._offset)),
                 self.process(sql.literal(select._limit)))
         return text
 
-    def visit_column(self, column, result_map=None, include_table=True, **kwargs):
+    def visit_column(
```

Comment:
The linter that runs has the value set to 120 for line length, this doesn't need to be broken up like this.